src/shared/components/palette/payment/case-payment-history-viewer-field.html

```diff
@@ -4,8 +4,5 @@
     [BULKSCAN_API_ROOT]="getPayBulkScanBaseURL()"
     [SELECTED_OPTION]="'CCDorException'"
     [ISBSENABLE]="'true'"
-    [LOGGEDINUSEREMAIL]="getUserEmail()"
```

Comment:
Ahh we need to restore these lines. 😩

Reply:
I merged Release branch into this branch to resolve conflicts and these lines do not exist on Release branch too, so we are safe :)